packages/bratus-app/src/api/index.js

```diff
@@ -2,8 +2,13 @@ export const getParsedData = () => {
   return fetch('http://localhost:4444/parsedData').then((res) => res.json());
 };
 
-export const recompile = () => {
+export const recompile = (input) => {
```

Comment:
Should we give it a more meaningful name like displaySubtree, or triggerRecompileForView or something nice?
Because it's more than recompiling.